tests/agent/test_agent.py

```diff
@@ -538,7 +538,6 @@ def agent_connect(self):
         assert False, "Failed to connect to health check"
 
     assert resp.status_code == 200
-    assert resp.json() == {}
```

Comment:
Interestingly (to me), `resp` was a normal request object, but `resp.json()` was returning a `MagicMock`